test/common/utils/accessible-menus.e2e.spec.js

```diff
@@ -0,0 +1,228 @@
+const E2eHelpers = require('../../e2e/e2e-helpers');
+const Timeouts = require('../../e2e/timeouts.js');
+
+const exploreButton = '#vetnav-menu button[aria-controls="vetnav-explore"]';
+const benefitsButton = '#vetnav-menu button[aria-controls="vetnav-benefits"]';
+const facilitiesLink = '#vetnav-menu a[href="/facilities/"]';
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const { SPACE, ENTER, UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW, ESCAPE } = client.Keys;
+
+    client
+      .url(`${E2eHelpers.baseUrl}/`);
+
+    E2eHelpers.overrideSmoothScrolling(client);
+
+    // landing page
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      // Make sure everything is where we think it should be
+      .waitForElementVisible(exploreButton, Timeouts.slow)
+      .waitForElementVisible(benefitsButton, Timeouts.slow)
+      .waitForElementVisible(facilitiesLink, Timeouts.slow)
+      // do not run 'wcag2a' rules because of open aXe bug https://github.com/dequelabs/axe-core/issues/214
+      .axeCheck('.main');
+
+
+    // --------------------- //
+    // --- Menubar tests --- //
+    // --------------------- //
+
+    // Explore benefits menu items
+    const firstMenuItem = 'button[aria-controls="vetnav-disability"]';
+    const secondMenuItem = 'button[aria-controls="vetnav-healthcare"]';
+    const lastMenuItem = 'button[aria-controls="vetnav-burials"]';
+
+    const testOpenExploreMenu = (key, focusedOn, keyName) => {
```

Comment:
Would it make sense to move this to a helper file?

Reply:
We could, but I think it'll only be used in this one test, so it seemed a bit extreme to pull it out into a helper file.